kodilla-google-selenium/src/main/java/pages/GoogleResultsFluke.java

```diff
@@ -0,0 +1,24 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.List;
+
+public class GoogleResultsFluke extends AbstractPage {
+
+    @FindBy(css = "div[class='g']")
+    private List<WebElement> resultsFluke;
+
+    public GoogleResultsFluke(WebDriver driver) {
+        super(driver);
+        PageFactory.initElements(this.driver, this);
+    }
+
+    public void iSeeResultsByFluke() {
```

Comment:
a co to za nazwa?

Reply:
Fluke - to Szczęśliwy traf

Reply:
kod jest z treści modułu. Dodałam do niego w GoogleSearch wyszukiwanie przez "Szczęśliwy traf". W nowej klasie GoogleResultsFluke jest metoda iSeeResultsByFluke.